distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogValidation.java

```diff
@@ -0,0 +1,1124 @@
+/**
```

Comment:
This file was created so I could pull out the 16 tests in TestCatalog that were starting/stopping the sample-validator each time. Since there is almost no overhead now for a new class it is faster to just start the feature once for all 16 of the tests instead of toggling it for each one.